src/com/aca/narekhakobyan/homework/items/functionality/format/FormatVideo.java

```diff
@@ -0,0 +1,5 @@
+package com.aca.narekhakobyan.homework.items.functionality.format;
+
+public enum FormatVideo {
```

Comment:
Why it is still here?

Reply:
You mean that we will not have any other Video format? If not, I can put it in Video type gadgets as a final Strings